c10/core/DispatchKeySet.cpp

```diff
@@ -118,6 +116,9 @@ DispatchKeySet getBackendKeySetFromAutograd(DispatchKey t) {
       return DispatchKeySet(DispatchKey::PrivateUse2);
     case DispatchKey::AutogradPrivateUse3:
       return DispatchKeySet(DispatchKey::PrivateUse3);
+    case DispatchKey::AutogradNestedTensor:
+      return DispatchKeySet(
+          {DispatchKey::NestedTensorCPU, DispatchKey::NestedTensorCUDA});
```

Comment:
This technically works, but it might be more robust to do:
```
return DispatchKeySet(DispatchKey::NestedTensor) | DispatchKeySet(DispatchKeySet::RAW, full_backend_mask);
```

Since NestedTensor technically supports other backends aside from CPU/CUDA (although I don't think there are implemntations for anything except for CPU/CUDA today).

Reply:
Ahhh that makes sense will change